tests/unit/api/v1/settings/test_get_settings.py

```diff
@@ -80,7 +81,7 @@ async def test_api_status_with_huggingface_info(self, async_client: AsyncClient)
                     "space_persistant_storage_enabled": True,
                 }
 
-    async def test_api_status_with_huggingface_info_and_not_running_on_huggingface(self, async_client: AsyncClient):
+    async def test_get_settings_for_huggingface_settings_not_running_on_huggingface(self, async_client: AsyncClient):
```

Comment:
this will return an empty dictionary, right?

Reply:
This will return the following:

```
{
  "argilla": {}
}
```

Attribute `huggingface` will not be present in this case.